utils/postgres_connector.py

```diff
@@ -0,0 +1,262 @@
+import psycopg2
+import pandas as pd
+
+
+class PostgresConnector:
```

Comment:
Rename class to a more descriptive name that includes insigths about aggregated performance, etl, transformation, etc.